src/OctoshiftCLI.IntegrationTests/AdoToGithub.cs

```diff
@@ -36,7 +36,7 @@ public AdoToGithub(ITestOutputHelper output)
 
             var githubToken = Environment.GetEnvironmentVariable("GHEC_PAT");
             _githubHttpClient = new HttpClient();
-            var githubClient = new GithubClient(logger, _githubHttpClient, new VersionChecker(_versionClient, logger), new RetryPolicy(logger), githubToken);
+            var githubClient = new GithubClient(logger, _githubHttpClient, new VersionChecker(_versionClient, logger), new RetryPolicy(logger), null, githubToken);
```

Comment:
wouldn't it be better to just pass in a `DateTimeProvider` instead of `null` ? it can be simply and inline instantiation.